scheduler/scheduler-server/src/main/java/org/ow2/proactive/scheduler/core/SchedulingMethodImpl.java

```diff
@@ -653,16 +653,24 @@ protected boolean createExecution(NodeSet nodeSet, Node node, InternalJob job, I
 
                     tlogger.debug(task.getId(), "deploying");
 
-                    finalizeStarting(job, task, node, launcher);
+                    getDBManager().registerUpcomingTaskStartedPersistence(job, task);
+
+                    String terminateNotificationURL = PAActiveObject.getActiveObjectNode(terminateNotification)
```

Comment:
You should rather use the name terminateNotificationNodeURL or  terminateNotificationAONodeURL everywhere, otherwise it can be confused to the url of an active oject

Reply:
done